StaticAnalyzer/views/ios/binary_analysis.py

```diff
@@ -358,21 +358,21 @@ def class_dump(tools_dir, bin_path, app_dir, bin_type):
         if platform.system() == 'Darwin':
             logger.info('Dumping classes')
             if bin_type == 'Swift':
-                logger.info('Running class-dump-swift aganst binary')
+                logger.info('Running class-dump-swift against binary')
                 if (len(settings.CLASSDUMP_SWIFT_BINARY) > 0
                         and is_file_exists(settings.CLASSDUMP_SWIFT_BINARY)):
                     class_dump_bin = settings.CLASSDUMP_SWIFT_BINARY
                 else:
                     class_dump_bin = os.path.join(
                         tools_dir, 'class-dump-swift')
             else:
-                logger.info('Running class-dump-z aganst binary')
+                logger.info('Running class-dump-z against binary')
                 if (len(settings.CLASSDUMPZ_BINARY) > 0
                         and is_file_exists(settings.CLASSDUMPZ_BINARY)):
                     class_dump_bin = settings.CLASSDUMPZ_BINARY
                 else:
                     class_dump_bin = os.path.join(tools_dir, 'class-dump-z')
-            os.chmod(class_dump_bin, 0o777)
+            os.chmod(class_dump_bin, 0o744)
```

Comment:
Here too

Reply:
in my test yes but it means that the user who do the install is the same who launch MobSF 
774 can be a safe alternative